zerver/tests/test_realm.py

```diff
@@ -152,6 +152,27 @@ def test_do_deactivate_realm_on_deactived_realm(self):
         do_deactivate_realm(realm)
         self.assertTrue(realm.deactivated)
 
+    def test_change_notifications_stream(self):
+        # type: () -> None
+        new_notif_stream_id = 4
+        realm = get_realm('zulip')
+        # we need an admin user.
```

Comment:
Minor but I think this call to get_realm can be removed. Also capitalize the starting word of comment.